setup.sh

```diff
@@ -3,6 +3,14 @@
 # Setup network configuration for a CentOS 6.6 virtual machine on VirtualBox
 # Please see directions at https://github.com/enterprisemediawiki/meza1
 
+# Get host-only IP address
+while [ -z "$ipaddr" ]
```

Comment:
You can just hit enter. The IP address you appear to be attempting to enter is in `install.sh`. This is `setup.sh` for getting your VirtualBox environment setup.